process-engine/bpmn-studio
# ✨ Add Active Cronjobs Table to Dashboard

## Changes

1. Add 'Active Cronjobs'-Table to Dashboard
2. Fix Hiding Process List if Request fails

## Issues

Closes #1546 

PR: #1556

## How to test the changes

- Create some diagrams with cycle start events
- Deploy the diagrams
- Go to dashboard
- **Notice that there will be a table for the active cronjobs**
